src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -117,6 +118,30 @@ export const NewProposal = ({ navigation, data, query }) => {
     setValue('tagList', filterData.toString());
   }, [tags]);
 
+  const uploadImage = async (imageData) => {
+    let imageAttributes = null;
+    const imageUploadData = await uploadAttachment(imageData, 'image');
+
+    if (imageUploadData.status === 200) {
+      const { filename, cached_attachment } = JSON.parse(imageUploadData.body);
+
+      imageAttributes = {
+        title: filename,
+        cachedAttachment: cached_attachment
+      };
```

Comment:
what do you say about:

```
const { filename: title, cached_attachment: cachedAttachment } = JSON.parse(imageUploadData.body);

imageAttributes = {
  title,
  cachedAttachment
};
```

Reply:
0bc389056155a1878daf189d621c7353bcc70657